librz/core/cfile.c

```diff
@@ -665,15 +665,17 @@ static int rz_core_file_do_load_for_debug(RzCore *r, ut64 baseaddr, const char *
 	}
 #endif
 	int fd = cf ? cf->fd : -1;
+
 	RzBinOptions opt;
-	rz_bin_options_init(&opt, fd, baseaddr, UT64_MAX, false, false);
+	rz_bin_options_init(&opt, fd, baseaddr, UT64_MAX, false, rz_config_get_i(r->config, "bin.rawstr"));
```

Comment:
```suggestion
	rz_bin_options_init(&opt, fd, baseaddr, UT64_MAX, false, false);
```
I would keep old behaviour (and also few lines below), unless this is carefully tested and we make sure the reason for doing as it was it not right/relevant any more.

Reply:
I revert the behavior.